src/Google/Ads/GoogleAds/Util/Testing/Google/Ads/GoogleAds/Util/Testing/TestCase.php

```diff
@@ -75,7 +75,17 @@ public function setDescription($var)
      */
     public function getOriginalResource()
     {
-        return $this->original_resource;
```

Comment:
This does not seem to be optional in the proto, should not the code stay the same as before?

I noticed the same thing happens with other fields in TestCase, Resource, Foo and Bar.

Reply:
This is what the protoc generates. If you have concerns, let's discuss with the protobuf team.
My two-cents are that it's not quite different in PHP.
If the variable is unset and you get the variable as is, it's returned as null.

Reply:
LGTM